CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.0 (Unreleased)
```

Comment:
A merge into master implies that we're ready to publish. There are a couple of enhancements that are in here that we put in as workarounds for Fiji. We made them a 4.0.0-alpha package (currently what's in package.json) anticipating that there would be breaking changes as part of the refactoring work.

We ended up detaching from shared-auth altogether on the React side and I anticipate we will probably do the same when we get to RN. So these couple of changes can/should get published as a minor version instead (3.8.0)